pallets/loans/src/lib.rs

```diff
@@ -781,13 +766,53 @@ pub mod pallet {
 
 			Ok(Some(T::WeightInfo::update_portfolio_valuation(count)).into())
 		}
+
+		/// Transfer debt from one loan to another loan,
+		/// repaying from the first loan and borrowing the same amount from the
+		/// second loan. `from_loan_id` is the loan used to repay.
+		/// `to_loan_id` is the loan used to borrow.
+		/// The repaid and borrow amount must match.
+		#[pallet::weight(10_000_000)] //TODO
+		#[pallet::call_index(11)]
+		pub fn transfer_debt(
+			origin: OriginFor<T>,
+			pool_id: T::PoolId,
+			from_loan_id: T::LoanId,
+			to_loan_id: T::LoanId,
+			repaid_amount: RepaidPricingAmount<T>,
+			borrow_amount: PricingAmount<T>,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+
+			ensure!(
+				from_loan_id != to_loan_id,
+				Error::<T>::TransferDebtToSameLoan
+			);
```

Comment:
Initially, I thought it was needed, but I think it's not and can be removed.
cc @offerijns

Reply:
I mean, a borrower can always borrow from the same loan to repay it. So, why forbid it here?

Reply:
@hieronx I've seen you've resolved this, does it mean you agree to remove it?